source/shadow/ConfigShadow.h

```diff
@@ -0,0 +1,209 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef AWS_IOT_DEVICE_CLIENT_CONFIGSHADOW_H
+#    define AWS_IOT_DEVICE_CLIENT_CONFIGSHADOW_H
+
+#endif // AWS_IOT_DEVICE_CLIENT_CONFIGSHADOW_H
```

Comment:
This line of code should be at the bottom of header file. Update it.

Reply:
updated